package.json

```diff
@@ -23,7 +23,7 @@
     "method-override": "2.3.3",
     "moment": "2.10.3",
     "passport": "0.2.2",
-    "passport-saml": "0.5.3",
```

Comment:
This'll need to be tested since IIRC newer versions of `passport-saml` had some issues with ADFS 2.0 - we don't want to fix one SSO provider only to break another. Will give it a spin in a cluster tomorrow.